src/features/projects/components/projectDetails/ProjectInfo.tsx

```diff
@@ -262,7 +312,7 @@ function ProjectInfo({ project }: Props): ReactElement {
             )}
 
             <div className={styles.infoText}>
-              {project.metadata.siteOwnerName} since{' '}
+              {project?.metadata?.siteOwnerName} since{' '}
```

Comment:
In scenario a> If siteOwnerName is null and acquisitionYear is not null 

"Since" should start with a capital "S".

(This does not apply to scenario b.)